src/NServiceBus.Extensions.DependencyInjection/Guard.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace NServiceBus.Extensions.DependencyInjection
```

Comment:
```suggestion
﻿namespace NServiceBus
```

See https://github.com/Particular/PlatformArchitecture/blob/master/principles/cross-cutting-concepts.md#nservicebus-rules